scripts/dev_setup.sh

```diff
@@ -892,6 +893,9 @@ while getopts "btoprvydaPJh:i:n" arg; do
   n)
     OPT_DIR="true"
     ;;
+  k)
```

Comment:
Could you also update `usage()` to describe what the `-k` flag is used for?

Reply:
The current dev_setup change is actually a hack... (I'm surprised you didn't hate it lol, because below it's even trying to `--break-system-packages`)

I could not figure out how to use existing dockerfile templates for pepper service, and the current dockerfile for pepper service is quick and dirty.

I will figure out the right way to docker build and revert this part. But if you are ok with the hack, i guess i can keep using the hack until i figure out?

Reply:
I think it's fine if this is just a quick hack

How is this image going to be built? If we want to use the GitHub workflows to automate this we probably do want to do something similar to the other Dockerfiles for caching